lib/gooddata/models/schedule.rb

```diff
@@ -67,15 +67,18 @@ def create(process_id, trigger, executable, options = {})
 
         fail 'Process ID has to be provided' if process_id.blank?
 
-        is_dataload_process = Process[process_id, project: project, client: c].type == :dataload
+        process = Process[process_id, project: project, client: c]
+        is_dataload_process = process.type == :dataload
 
         if is_dataload_process
           dataload_datasets = options[:dataload_datasets] || options['GDC_DATALOAD_DATASETS']
           dataload_datasets = '[]' unless dataload_datasets
 
           de_synchronize_all = options[:de_synchronize_all] || options['GDC_DE_SYNCHRONIZE_ALL']
         else
-          fail 'Executable has to be provided' if executable.blank?
+          lcm_component = process.type == :lcm
```

Comment:
what about processes of other component types (etl..) that also do not have an executable?